vtkext/private/module/vtkF3DRenderer.cxx

```diff
@@ -1315,6 +1315,7 @@ void vtkF3DRenderer::FillCheatSheetHotkeys(std::stringstream& cheatSheetText)
   cheatSheetText << std::fixed;
   cheatSheetText << " L: Light (increase, shift+L: decrease) [" << this->LightIntensity << "]"
                  << " \n";
+  cheatSheetText << " F1: Enable Backface Culling " << (this->BackfaceCullingVisible ? "[ON]" : "[OFF]") << "\n";
```

Comment:
lets not add a interactiive binding for this, I think it is too specific.

Reply:
Fixed!